css-three-column-layout/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  font-family: Lato, sans-serif;
+}
+
+section {
+  background-color: #1a96d9;
+  color: #fff;
+  padding: 40px 0;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
```

Comment:
Instead of centering everything in the row, try making a container that sets a narrower max-width. Then you can make column-3 actually 1/3rd instead of 1/4th wide.